Server/02-start.ps1

```diff
@@ -15,14 +15,18 @@ $OctopusServerContainer= $ProjectName+"_octopus_1";
 $env:OCTOPUS_VERSION=Get-ImageVersion $OctopusVersion $OSVersion;
 $env:OCTOPUS_SERVER_REPO_SUFFIX="-prerelease"
 $env:SERVERCORE_VERSION=$OSVersion
-$env:OCTOPUS_SKIP_IMPORT_VERSION_CHECK="true"
+$env:OCTOPUS_SKIP_IMPORT_VERSION_CHECK="false"
```

Comment:
I think we still need this, for tentacle at least.

For server, we know what version we are launching, so we can use `00-hacktheimportfiles.ps1` to set the version.
For Tentacle, we ask for `latest`, so we dont know what version we're getting - we aren't able to modify the version in the import files, so it will go :boom: